src/dexnav.c

```diff
@@ -2314,27 +2314,42 @@ static void DexNavPopulateEncounterList(void)
 
 static void RegisterSpecies(u16 species, u8 taskId)
 {
-	//Species was valid
-	DexNavDisplaySpeciesData();
+	// Disable selection if toggled again
```

Comment:
nit: some of the comments are done like //This comment  and others are done like // This comment.

no need to change, just pointing out inconsistency

Reply:
Good point.  The ones without spaces are original code from the engine, and the ones with are my code convention.  I may change it someday ;)